app/scripts/proactive/rest/studio-client.js

```diff
@@ -140,25 +179,43 @@ define(
                     $.ajax({
+                        async: false,
                         type: "GET",
-                        url: config.restApiUrl + "/connected",
+                        url: JSON.parse(localStorage['restUrl']) + '/common/permissions/portals/studio',
                         beforeSend: function(xhr) {
-                            xhr.setRequestHeader('sessionid', localStorage['pa.session'])
+                            xhr.setRequestHeader('sessionid', localStorage['pa.session'],
+                                                'Content-Type', 'application/json')
                         },
-                        success: function(data) {
-                            if (data) {
-                                console.log("Connected to the studio", data)
-                                success()
+                        success: function(response) {
+                            if(response){
+                                $.ajax({
+                                    type: "GET",
+                                    url: config.restApiUrl + "/connected",
+                                    beforeSend: function(xhr) {
+                                        xhr.setRequestHeader('sessionid', localStorage['pa.session'])
+                                    },
+                                    success: function(data) {
+                                        if (data) {
+                                            console.log("Connected to the studio", data)
+                                            success()
+                                        } else {
+                                            console.log("Not connected to the studio", data)
+                                            localStorage.removeItem('pa.session');
+                                            fail()
+                                        }
+                                    },
+                                    error: function(data) {
+                                        console.log("Not connected to the studio", data)
+                                        localStorage.removeItem('pa.session')
+                                        fail()
+                                    }
+                                });
                             } else {
-                                console.log("Not connected to the studio", data)
-                                localStorage.removeItem('pa.session');
-                                fail()
+                                fail();
                             }
                         },
-                        error: function(data) {
-                            console.log("Not connected to the studio", data)
-                            localStorage.removeItem('pa.session')
-                            fail()
+                        error: function(response) {
+                            console.error("Failed to know the user permission", response)
```

Comment:
User is not authorized

Reply:
It's the case where the request failed, we don't know if he is authorized or not

Reply:
so: "Unknown User Permission"

Reply:
ok